core/src/main/java/nu/yona/server/goals/entities/TimeZoneGoal.java

```diff
@@ -43,4 +45,11 @@ public boolean isNoGoGoal()
 	{
 		return false;
 	}
+
+	@Override
+	public boolean isGoalAccomplished(DayActivity dayActivity)
+	{
+		// TODO: zones should be parsed? maybe in spread format
+		return true;
```

Comment:
That text format sounds good. We currently don't constrain the time zones to 15-minute boundaries. I'm writing a mail to Jan and Cornelis. I'll include that proposal. With that, we can internally make a translation to 96 booleans if we want. Note that we will need to store the original time spans too.